src/scss/modules/sidebar.scss

```diff
@@ -1,7 +1,8 @@
 .sidebar {
   border-left: 1px solid #{$black-90};
   overflow-y: auto;
-  padding: 16px 20px;
+  padding: 16px 12px;
+  scrollbar-gutter: stable both-edges;
```

Comment:
Also this property only started to be supported by Safari around 6 months ago. So there could be a good percentage of users who won't have this supported.

Nothing would break for them. But the right/left padding would tight at `12px`.